correios/models/builders.py

```diff
@@ -111,7 +111,10 @@ def build_post_info(self, data, user: User) -> PostInfo:
         )
         return post_info
 
-    def build_receipt(self, data) -> Receipt:
+    def build_receipt(self, data):
```

Comment:
dá pra manter o typing

```suggestion
    def build_receipt(self, data) -> Union[None, bool]:
```